DQM/include/DQM/HgcrocPulseTruth.h

```diff
@@ -0,0 +1,39 @@
+#ifndef _DQM_HGCROCPULSETRUTH_H_
+#define _DQM_HGCROCPULSETRUTH_H_
+
+/*~~~~~~~~~~~~~~~*/
+/*   Framework   */
+/*~~~~~~~~~~~~~~~*/
+#include "Framework/Configure/Parameters.h"
+#include "Framework/Event.h"
+#include "Framework/EventProcessor.h"
+#include "TGraph.h"
+#include "Tools/AnalysisUtils.h"
+
+namespace dqm {
+
+class HgcrocPulseTruth : public framework::Analyzer {
+ public:
+  HgcrocPulseTruth(const std::string& name, framework::Process& process)
+      : framework::Analyzer(name, process) {}
+
+  ~HgcrocPulseTruth() {}
```

Comment:
default this

Reply:
The constructor can not be defaulted here. There needs to be some function body after the `framework::Analyzer(name, process)` initialization list, even if it just a couple of empty curly-braces like here. 

The destructor could be defaulted though (by just removing that line).

Reply:
I meant the destructor indeed. Tho I thought to default here & then remove in the cxx, but I guess it's fine like you did it.